pwd_guess.py

```diff
@@ -580,6 +583,7 @@ class ModelDefaults(object):
     dropouts = False
     dropout_ratio = .25
     fuzzy_training_smoothing = False
+    tensorboard = True
```

Comment:
Does it make sense to default to False here? I would prefer to preserve the typical behavior for tests so that the unit tests don't make non-temporary files.